client/src/component/Footer.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+function Footer() {
+  return (
+  	<div className="App-footer" id="Footer">
+	  	<div className="container">
+		  	<div className="social">
+			  	<a href="mailto:general@hackmerced.com?subject=HackMerced Questions&amp;body=Dear HackMerced,">
+			  		<i className="fa fa-envelope"></i>
+			  	</a>
+			  	<a href="https://www.facebook.com/hackmerced/" target="_blank" rel="noopener noreferrer">
+			  		<i className="fa fa-facebook-square"></i>
+			  	</a>
+			  	<a href="https://www.instagram.com/hackmerced/">
+			  		<i className="fa fa-instagram"></i>
+			  	</a>
+			  	<a href="https://slack.com">
```

Comment:
```suggestion
			  	<a href="#">
```

So that it just refreshes the page